### ISSUE TITLE
Improved position syntax

### ISSUE THREAD

**Issue by dkrutsko** [MoOx/postcss-cssnext#270]
Would love to see some of the features from [Nib](//tj.github.io/nib) make it into cssnext. Particularly [Position mixins](//github.com/tj/nib/blob/master/lib/nib/positions.styl). I think [this](//www.npmjs.com/package/postcss-position-alt) plugin does something similar. Quick example:

```
#back-to-top {
  fixed: bottom right;
}

#back-to-top {
  position: fixed;
  bottom: 0;
  right: 0;
}
```

**Comment by MoOx** [comment]
cssnext only supports future proof CSS, based on CSS specification (drafts etc).
This might be implemented via #203.

**Comment by dkrutsko** [comment]
Ah makes sense, thanks for the clarification!
